profile.html

```diff
@@ -215,8 +215,10 @@ <h2 class="avatar__heading">Jacob Washington</h2>
       <i class="navigation__icon--grey fa fa-bell fa-lg"
         ><span class="sr-only">Bell</span></i
```

Comment:
Reading out the word "Bell" does not make this accessible enough. What does this icon really mean?